modules/n1ql/pages/n1ql-language-reference/subqueries.adoc

```diff
@@ -271,36 +347,75 @@ An implicit keyspace alias is not considered as an expression, as in the above e
 However, the nested paths are expressions.
 In the following example Q6C, the `from-term` has nested path `pass:c[`travel-sample`.geo]` as expression where `pass:c[`travel-sample`]` is referring to the implicit alias of the keyspace in the parent query.
 Hence this is a correlated subquery, and the result is `1` corresponds to the current document in the parent query.
+//ambigious reference error
 +
 [source#Q6C,json]
 ----
 Example Q6C:
 
 SELECT array_length((SELECT RAW t1.alt
                      FROM `travel-sample`.geo t1))
```

Comment:
```suggestion
SELECT array_length((SELECT RAW t1.alt
                     FROM airport.geo t1))
```

The last element (`airport`) in the nested keyspace identifier is the implicit alias.